### DISCUSSION TITLE
Can i filter with case insensitive

### DISCUSSION THREAD

**Discussion by guillermo2007** [OData/AspNetCoreOData#633] [General]
Hello everyone.

I have a problem. 
I have tried to filter (with contains) a query with some words, for example, dog, DOG or Dog and odata only find the exact word.
And i have de same problem when my object has a property that it is an object and i need to get access to properties.
Example: Car/Seat/Type. Car is a object with a property Seat that is a object with a property Type and the last is a string

Can i activate case insensitive?

Thanks

**Comment by vivekvenugopal** [comment]
Have you found solution to the problem. We are having the same issue
